.github/actions/create-jira-ticket/action.yml

```diff
@@ -14,320 +14,183 @@ inputs:
-        unset IFS
-        [ -n "${saved_IFS+set}" ] && { IFS=$saved_IFS; unset saved_IFS; }
-
-        for label in ${ticket_labels[@]}; do
-          response=$(curl \
-            --request PUT \
-            --url "${{ inputs.jira_base_url }}/rest/api/3/issue/${{ env.ticket_key }}" \
-            --user "${{ inputs.jira_user_email }}:${{ inputs.jira_api_token }}" \
-            --header 'Accept: application/json' \
-            --header 'Content-Type: application/json' \
-            --data "{ \"update\": { \"labels\": [ { \"add\": \"$label\" } ] } }"
-            )
-        done
-
-        ticket_description=$(curl --request GET \
-          --url "${{ inputs.jira_base_url }}/rest/api/3/issue/${{ env.ticket_key }}" \
-          --user "${{ inputs.jira_user_email }}:${{ inputs.jira_api_token }}" \
-          --header "Accept: application/json" | jq '.fields.description')
-
-        mapfile -t jobs_failed < <(gh run view ${{ github.run_id }} --json jobs -q '.jobs[] | select(.conclusion == "failure") | .name')
-        echo "[DEBUG] - jobs failed for component ${FAILED_COMPONENTS[index]}: $jobs_failed"
-
-        new_list_of_failed_jobs=$(for job in "${jobs_failed[@]}"; do
-          cat <<EOF
-        {
-          "type": "listItem",
-          "content": [
-            {
-              "type": "paragraph",
-              "content": [
-                {
-                  "type": "text",
-                  "text": "$job"
-                }
-              ]
+        TICKET_REASON_FOR_CREATION: ${{ inputs.ticket_reason_for_creation }}
+        MODULE_NAME: ${{ inputs.module_name }}
+        PR_NUMBER: ${{ github.event.pull_request.number }}
+        PR_URL: ${{ github.event.pull_request.html_url }}
+        REF_NAME: ${{ github.ref_name }}
+      id: get_context
+      with:
+        script: |
+          const fs = require('fs');
+          const { execSync } = require('child_process');
+          const dayjs = require('dayjs');
+          const isoWeek = require('dayjs/plugin/isoWeek');
+          dayjs.extend(isoWeek);
+
+          const parent_epic_id = 206242
+          const parent_epic_key = "MON-151547"
+          const JSON_TEMPLATE_FILE = "./.github/actions/create-jira-ticket/nightly-ticket-template.json";
+          const date = dayjs().format('YYYY-MM-DD');
+          const path = './ticket-payload.json';
+          let contents;
+          let data, ticket_summary, ticket_squad_id, ticket_board_id, squad_name, ticket_labels, current_sprint, ticket_data, data_payload, ticket_key;
+
+          let failed_jobs = execSync(
+            `gh run view ${{ github.run_id }} --json jobs -q '.jobs[] | select(.conclusion == "failure") | .name'`,
+            { encoding: 'utf8' }
+          );
+
+          failed_jobs = failed_jobs
+            .split('\n')
+            .map(line => line.trim())
+            .filter(Boolean);
+
+          console.log(failed_jobs);
+
+          for (const failed_job of failed_jobs) {
```

Comment:
Is it possible to group tickets by global workflow fail ?
example:
* e2e tests together
* api tests together
* legacy e2e tests together
It will avoid to have tons of tickets if lot of e2e tests fail